src/python/gem5py_m5.cc

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2019 ARM Limited
+ * All rights reserved
+ *
```

Comment:
Not a blocker but you're not giving yourself credit here. If based on "gem5py.cc" you've had to hack quite a bit of this to get it working.